common/Cargo.toml

```diff
@@ -9,7 +9,6 @@ default = []
 std = [
     "failure/std",
     "mc-util-serial/std",
-    "serde/std",
```

Comment:
are you sure this is redundant?

Reply:
`mc-util-serial/std` enables it.

Reply:
i think at this revision, when loggers => std, you still need `serde/std` to be implied by the std feature

Reply:
oh i see

yeah do whatever you like here.